src/script/index.js

```diff
@@ -96,7 +96,8 @@ window.onload = function () {
 
 const buildInfo = {
 	version: '__VERSION__',
-	apiPath: '__API_PATH__',
+	apiPath: 'http://127.0.0.1:5000/',
```

Comment:
How does this get used?  We probably want a TODO to wire this up when we move the backend to the bluebird-chem repo (currently bluebird-standardizer)